src/libraries/System.Private.CoreLib/src/System/Diagnostics/StackTrace.cs

```diff
@@ -201,7 +201,9 @@ internal string ToString(TraceFormat traceFormat)
 
         internal void ToString(TraceFormat traceFormat, StringBuilder sb)
         {
-            string word_At = SR.Word_At;
+            // Adding a default string for "at" in case we are using Resource Keys instead of resources
+            // as this is a special case and we don't want to have "Word_At" on stack traces.
+            string word_At = SR.GetResourceString(nameof(SR.Word_At), defaultString: "at");
             string inFileLineNum = SR.StackTrace_InFileLineNumber;
```

Comment:
Question: Should we do the same for the InFileNumber?

Reply:
I would think so